src/raiden_libs/matrix.py

```diff
@@ -274,3 +272,51 @@ def _handle_message(self, room: Any, event: dict) -> bool:
             self.message_received_callback(message)
 
         return True
+
+    def join_global_rooms(
```

Comment:
Currently most of these functions reside in `raiden.network.transport.utils`. I'd put this one there as well and add some tests.

Reply:
I think it is confusing to have `join_global_room` and `join_global_rooms` in `raiden.network.transport.utils`. As we would like to keep both and `join_global_rooms` is specific for the services, I would leave it here.

Reply:
My point is that the services have minimal matrix related code and no integration tests for matrix.
Therefore, I think it's better to move this to the other utils (and change the name to clarify) and add some tests with the others.

Reply:
The function is supposed to be used _only_ in the services, just like the matrix_listener it belongs to. Is it okay If we just add the tests in raiden.matrix like with the rest of the services stuff but leave the function where it is used?

Reply:
And here we are: https://github.com/raiden-network/raiden-services/issues/630#issuecomment-554373344